<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImage</class>
 <widget class="QDialog" name="DialogImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1202</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="lv_image">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>331</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>471</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>显示</string>
   </property>
   <widget class="QLabel" name="lb_img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>431</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ptn_addimage">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>510</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ptn_search">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>40</y>
     <width>311</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>属性</string>
   </property>
   <widget class="QCheckBox" name="cb_RGB">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable RGB Dither</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Alpha">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Alpha Dither</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_Color">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>191</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Color key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_from_color">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>colorkey(from)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_end_color">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>colorkey(end)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>291</y>
      <width>88</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha Value</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>96</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pixel format</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combAlpha">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Alpha Const ZERO</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alpha Const ONE</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alpha From SRC</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="combPixel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ARGB8888</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ARGB4444</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB565</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RGB555</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BLINK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ARGB1555</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确定修改</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_ID_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ptn_mirror">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>510</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>图片mirror</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_count">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>一共0张</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptn_delete">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_tips">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>510</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>tips</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_from_color_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>480</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>_mirror</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>480</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>后缀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptn_clone">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>510</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>图片克隆</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
